ui/src/main/res/layout/tunnel_list_item.xml

```diff
@@ -49,6 +49,20 @@
             android:textAppearance="?attr/textAppearanceBodyLarge"
             tools:text="@sample/interface_names.json/names/names/name" />
 
+        <com.google.android.material.checkbox.MaterialCheckBox
+            android:id="@+id/tunnel_shortcut_toggle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignBaseline="@id/tunnel_switch"
+            android:layout_toStartOf="@id/tunnel_switch"
+            android:button="@drawable/ic_action_shortcut"
+            android:checked="@{item.hasShortcut == true}"
+            android:nextFocusRight="@+id/tunnel_switch"
+            android:onCheckedChanged="@{fragment::setShortcutState}"
+            android:tooltipText="@string/tooltip_tunnel_shortcut"
+            app:buttonTint="?attr/colorSecondary"
+            tools:checked="@sample/interface_names.json/names/shortcut/shortcut" />
```

Comment:
I wonder if this can be more hidden or less prominent or something. This is not something most users want.

Another approach -- and the one we use on iOS -- which I think might be better is to just add toggle shortcuts for the last N used tunnels.